ddmrp_exclude_moves_adu_calc_sales/views/sale_order_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form - ddmrp_exclude_moves_adu_calc_sales</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/list" position="inside">
+                <field
+                    name="exclude_from_adu"
+                    optional="hide"
+                    widget="boolean_toggle"
+                    attrs="{'column_invisible': [('parent.state', 'not in', ['sale', 'done'])]}"
```

Comment:
I think this is not working as expected:

![image](https://github.com/user-attachments/assets/2475f131-97cb-494f-aa68-545932b1a506)

Reply:
Fixed! This change need to be fixed in version 17.0 too